contrib/board_files/example.conf

```diff
@@ -0,0 +1,20 @@
+# A board config file consists of two required sections, the "board" and "GPIO"
+# Lines starting with "#" are ignored
+
+[board]
+# The dtfile is the file probed to determine if current board matches this
+# board config. Change this value if you system requires a different file for
+# board identification:
+# dtfile = /proc/device-tree/model
+
+# The model value is compared to the contents of the dtfile. If these match,
+# then this board config will be selected.
```

Comment:
can you state that the _whole_ of the contents of the file have to match, and not just a portion of the file